dlgr/griduniverse/experiment.py

```diff
@@ -885,16 +886,24 @@ def spawn_player(self, id=None, **kwargs):
         self._start_if_ready()
         return player
 
-    def _random_empty_position(self, item_id):
+    def _random_empty_position(self, item_id=None, player=False):
         """Select an empty cell at random, using the configured probability
         distribution."""
         rows = self.rows
         columns = self.columns
         empty_cell = False
+        if item_id:
+            prob_func = self.item_config[item_id]["probability_function"]
```

Comment:
Doesn't this need to be `getattr(distributions, self.item_config[item_id]["probability_function"])` (the config value is just a string)?

Reply:
Oh, I see... you've already created a map to the actual functions earlier. 👍🏼